monkeys/build.gradle.kts

```diff
@@ -68,6 +68,23 @@ sourceSets {
         }
     }
 
+    val test by getting {
+        dependencies {
+            implementation(kotlin("test"))
+
+            // coroutines
+            implementation(libs.coroutines.test)
+            implementation(libs.turbine)
+
+            // mocking
+            implementation(libs.mockative.runtime)
+            implementation(libs.okio.test)
+            implementation(libs.settings.kmpTest)
+
+            implementation("io.mockk:mockk:1.13.5")
```

Comment:
This should be moved to the version catalog (`libs.versions.toml`). This is the new standard, has better IDE support, etc.